beesdoo_shift_welcome_screen/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+beesdoo_shift_welcome_screen.access_beesdoo_shift_welcome,access_beesdoo_shift_welcome,beesdoo_shift_welcome_screen.model_beesdoo_shift_welcome,base.group_user,1,1,1,1
```

Comment:
in the `id` and the `model_id:id column`, no need to prefix with the module name as it is defined in this very module.